app/presenters/content_item_presenter.rb

```diff
@@ -1,5 +1,6 @@
 class ContentItemPresenter
   include Withdrawable
+  include BreadcrumbHelper
```

Comment:
I don't _think_ this needs to be explicitly included, the other helpers aren't, but are available to the view.